diff --git a/contrib/windows-cmake/CMakeLists.txt b/contrib/windows-cmake/CMakeLists.txt
index 8a7610fc7..d32fc34a8 100644
--- a/contrib/windows-cmake/CMakeLists.txt
+++ b/contrib/windows-cmake/CMakeLists.txt
@@ -59,6 +59,7 @@ elseif (CMAKE_SYSTEM_PROCESSOR MATCHES "(^x86$|i.86)")
 else()
     set(HWLOC_HAVE_X86_CPUID 0)
 endif()
+message(STATUS "CMAKE_SYSTEM_PROCESSOR = ${CMAKE_SYSTEM_PROCESSOR}\nHWLOC_HAVE_X86_CPUID = ${HWLOC_HAVE_X86_CPUID}")
 
 check_c_source_compiles("int main(void) {int cpuinfo[4]; __cpuidex(cpuinfo,0,0); return 0;}"
 HWLOC_HAVE_MSVC_CPUIDEX
